hack/tilt/kubetail.yaml

```diff
@@ -45,9 +45,92 @@ data:
           enabled: true
           hide-health-checks: true
       tls:
-        enabled: false
-        cert-file:
-        key-file:
+        enabled: true
+        cert-file: /etc/kubetail/tls/tls.crt
+        key-file: /etc/kubetail/tls/tls.key
+  # TLS certificate and key generated using - openssl req -x509 -newkey rsa:4096 -keyout tls.key -out tls.crt -days 365 -nodes -subj "/CN=localhost"
+  tls.crt: |
```

Comment:
Let's split this out into a `Secret` of type tls (https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets)